python/examples/example_constant_species/ExampleEquilibrationLogic.ipynb

```diff
@@ -0,0 +1,587 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# AMICI documentation example of the steady state solver logic\n",
+    "\n",
+    "This is an example to document the internal logic of the steady state solver, which is used in preequilibration and postequilibration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Steady states of dynamical system\n",
+    "\n",
+    "Not every dynamical system needs to run into a steady state. Instead, it may exhibit \n",
+    " * continuous growth, e.g., $$\\dot{x} = x, \\quad x_0 = 1$$\n",
+    " * a finite-time blow up, e.g., $$\\dot{x} = x^2, \\quad x_0 = 1$$\n",
+    " * oscillations, e.g., $$\\ddot{x} = -x, \\quad x_0 = 1$$\n",
+    " * chaotic beviour, e.g., the Lorentz attractor\n",
+    " \n",
+    "If the considered dynamical system has a steady state for positive times, then integrating the ODE long enough will bring equilibrate the system to this steady state. However, this may be computational more demanding than other approaches and may fail, if the maximum number of integration steps is exceeded before reaching the steady state.\n",
```

Comment:
```suggestion
    "If the considered dynamical system has a steady state for positive times, then integrating the ODE long enough will equilibrate the system to this steady state. However, this may be computationally more demanding than other approaches and may fail, if the maximum number of integration steps is exceeded before reaching the steady state.\n",
```